### ISSUE TITLE
The information about upgrading to the business package etc doesn't seem to work and is out of date

### ISSUE THREAD

**Issue by GrabzIt** [riganti/dotvvm-docs#125]
This is regarding: https://www.dotvvm.com/docs/3.0/pages/dotvvm-for-visual-studio/dotvvm-private-nuget-feed#sign-in-to-add-the-feed-automatically

The automatic feed registration doesn't work neither does the automatic install of the packages.

I got it working via the private nuget feed however I used the instructions here: 

https://docs.telerik.com/reporting/add-the-telerik-private-nuget-feed-to-visual-studio

Basically in the nuget settings option in the nuget manager. Add the URL as a new source then when the username and password is asked for enter it and choose the save password option.
